### PR TITLE
Update conf info

### PR DESCRIPTION
**Pull request by j8xixo12**
I have updated cyntx project introduction topic and discussion topic on the program page and project page, and I have also updated NASA TOPS training figure.

### CODE REVIEW

**Review by rockleona — APPROVED** [review]
LGTM!

File: app/program/page.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const Page = () => {
             <TalkInfoCard
               talk={{
                 title: "Introduction - cyntx",
```
Comment (YenPeiChen07) [inline app/program/page.tsx]: There is a typo to `cytnx` in the program timetable.
image

File: app/program/page.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ const Page = () => {
...
-                description: "",
+                description: "Cytnx: An Introduction to the Tensor Network Library for Quantum Physics",
                 room: "R716",
```
Comment (YenPeiChen07) [inline app/program/page.tsx:60]: Could you please also add the speaker name (`Ying-Jer Kao`) for Cytnx talk ?

**Review by YenPeiChen07 — COMMENTED** [review]
@j8xixo12, Thank you for updating the project information. It looks great. 

Could you please help fix the typo and add the speaker name for Cytnx talk?

File: app/program/page.tsx (hunk 2 above)
Comment (j8xixo12) [reply app/program/page.tsx:60]: Done !

File: app/program/page.tsx (hunk 1 above)
Comment (j8xixo12) [reply app/program/page.tsx]: Done !

**Review by YenPeiChen07 — APPROVED** [review]
LGTM.

**Pull request merged**